File: c++/worker.cpp
#include <string>
#include "worker.hpp"
#include <iostream>

using namespace std;

Worker::Worker(string _name, int _age){
    name = _name;
    age = _age;
}

Worker::~Worker() {
    cout << "Instance removed" << endl;
}

void Worker::receive_salay(){
    cout << name << "receive_salay"<< endl;
}

File: c#/worker.cs
using System;
public class Worker{
		protected string name;
		protected int age;
		protected int workerNum;
	
		public Worker(string _name, int _age, int _workerNum) {
			name = _name;
			age = _age;
			workerNum = _workerNum;
		}

        ~Worker() {
            System.Console.WriteLine("Instance removed.");
        }
}